src/handlers/RequestHandler.ts

```diff
@@ -37,6 +37,7 @@ export type DefaultRequestMultipartBody = Record<
 export type DefaultBodyType =
   | Record<string, any>
   | DefaultRequestMultipartBody
+  | FormData
```

Comment:
Formdata won't exist in node. I think we need to use a polyfill for both types and runtime.